src/Generators/ModelflakeGenerator.php

```diff
@@ -8,17 +8,41 @@
  * Class ModelflakeGenerator
  *
  * Custom implementation inspired by Twitter Snowflake.
- * Generate Snowflake IDs with a 7-bit prefix identifier for the model that the ID was generated for
```

Comment:
Why did you remove the "7-bit" description? I left that there to specify how this should be built